rbac/management/querysets.py

```diff
@@ -105,6 +105,8 @@ def _gather_group_querysets(request):
     ) or Group.platform_default_set().filter(tenant=public_tenant)
 
     username = request.query_params.get("username")
+    if not username and kwargs:
+        username = kwargs.get("principals")
```

Comment:
I made it "principals" plural mostly because the filter tied into the 'principals' field of the Group model. Though thinking further, I believe that multi-value filtering will work fine, so principals makes sense that way too! Up for a change if we want to make it singular, though.